boofuzz/__init__.py

```diff
@@ -643,8 +658,9 @@ def s_mirror(primitive_name, name=None):
     if primitive_name not in blocks.CURRENT.names:
         raise exception.SullyRuntimeError("CAN NOT ADD A MIRROR FOR A NON-EXIST PRIMITIVE CURRENTLY")
 
-    mirror = primitives.Mirror(primitive_name, blocks.CURRENT, name)
+    mirror = primitives.Mirror(primitive_name, blocks.CURRENT)
     blocks.CURRENT.push(mirror)
+    blocks.CURRENT.push(FuzzableWrapper(fuzz_object=mirror, name=name, fuzzable=True, default_value=None))
```

Comment:
Any special for `s_mirror`? There is an extra statement `blocks.CURRENT.push(mirror)`.

Reply:
Good catch, `blocks.CURRENT.push(mirror)` should go.